Source/WebCore/platform/ScrollableArea.h

```diff
@@ -215,6 +216,7 @@ class ScrollableArea : public CanMakeWeakPtr<ScrollableArea> {
     ScrollAnimator* existingScrollAnimator() const { return m_scrollAnimator.get(); }
 
     WEBCORE_EXPORT ScrollbarsController& scrollbarsController() const;
+    ScrollbarsController* rawScrollbarsController() const { return m_scrollbarsController.get(); }
```

Comment:
"raw" is unusual; we don't do that anywhere else. Use `existingScrollbarsController()`

Reply:
Fixed.